tests/apps/mip/test_config_mip.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from cg.exc import PedigreeConfigError
+from cg.apps.mip.confighandler import ConfigHandler
+
+
+def test_validate_config(valid_config):
+    """Test to validate a correct config"""
+    # GIVEN a correct config
+    # WHEN validating the config
+    errors = ConfigHandler.validate_config(valid_config)
```

Comment:
Would be nice with some newlines between paragraphs here in the tests for readability